merfi/backends/__init__.py

```diff
@@ -1,2 +1,4 @@
-import rpm_sign
-import gpg
+from __future__ import absolute_import
+
+import merfi.backends.rpm_sign
```

Comment:
doing this doesn't work?
```suggestion
from merfi.backends import rpm_sign
```

Reply:
Four years later ... yes